docs/modules/user/pages/quickstart.adoc

```diff
@@ -58,6 +58,46 @@ services:
     restart: unless-stopped
 ....
 
+Podman quadlet:
+
+[source,ini]
+....
+[Unit]
+Description=Podman Wolf Gamestreaming
```

Comment:
This unit implies you're using a system wide podman service. This will work if you have `podman.socket` active when you run this unit, but it would be best if you added the dependency explicitly.

```
[Unit]
Description=Podman Wolf Gamestreaming
Requires=podman.service
After=podman.service 
```